tests/run-tests.sh

```diff
@@ -443,6 +443,17 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is correctly updated 
+case ${distro} in
+    Ubuntu*) sudo apt-get -q --assume-no update;;
+    CentOS* | AlmaLinux*) 
+        yum update -y --setopt tsflags=test
```

Comment:
semicolons at the end of each line inside a case condition

Reply:
Oh! you also need to run it as sudo

Reply:
> semicolons at the end of each line inside a case condition

I think you only need the double semicolons at the end of the condition. No other semicolons at the end of each line withing the condition. That's how I found it in the examples.